src/components/TodoComponents/TodoList.js

```diff
@@ -6,7 +6,7 @@ export default class TodoList extends React.Component {
     return (
       <div>
         {this.props.list.map(i => {
-          return <Todo key={i.id} id={i.id} task={i.task} />;
+          return <Todo key={i.id} id={i.id} task={i.task} onClick={this.props.done} />;
```

Comment:
Here we want to put the `onClick` on a HTML element rather than a component. In this example we would add the `onClick` to the div container inside the `Todo` component. 👍

Reply:
Also, here we want to use pass the `id` of the `Todo` to the `.done` function like so: 
```javascript
onClick={() => this.props.done(this.props.id)}
```